### PR TITLE
including the 76X pileup Jet Id training 

### PR DESCRIPTION
**Pull request by jbrands**
This PR comes together with https://github.com/cms-data/RecoJets-JetProducers/pull/7
@kkousour and @ahinzmann 
Validation plots can be found here: http://www.hephy.at/user/jbrandstetter/PUId_validation76X/

### CODE REVIEW

File: RecoJets/JetProducers/python/PileupJetID_cfi.py [hunk 1]
```diff
@@ -7,9 +7,10 @@
 
 #_chsalgos_4x = cms.VPSet(full,   cutbased) 
 _chsalgos_5x = cms.VPSet(full_5x_chs,cutbased)
-_chsalgos_7x = cms.VPSet(full_74x_chs,cutbased)
+_chsalgos_74x = cms.VPSet(full_74x_chs,cutbased)
+_chsalgos_76x = cms.VPSet(full_76x_chs,cutbased)
 
-_stdalgos    = _chsalgos_7x
+_stdalgos    = _chsalgos_76x
```
Comment (slava77) [inline RecoJets/JetProducers/python/PileupJetID_cfi.py]: this will change 76X behavior in miniAOD and alike.
It is not allowed in 76X.

**Comment by slava77** [comment]
If I understood correctly , this PR will change miniAOD output.

If you want this PR as a recipe, it's fine.
It will not be merged in 76X (until a new miniAOD campaign) 
and the PR should probably be closed for now.
It may be more convenient for users if you can make changes from this PR configurable and preserve the default behavior to both match the release policy and deliver code that doesn't need recompilation for analysis purposes.

File: RecoJets/JetProducers/python/PileupJetID_cfi.py (hunk 1 above)
Comment (ahinzmann) [inline RecoJets/JetProducers/python/PileupJetID_cfi.py]: Yes, the 76X version of this PR is rather a placeholder for an analyses recipe.

**Comment by ahinzmann** [comment]
@jbrands Could you cherry-pick also this additional commit into the 76, 80 and 81 PRs?
https://github.com/cms-sw/cmssw/commit/d80860dc16a9828a0216c67ba61926a36e40a96f
It makes sure that people running the PU jet ID on MiniAOD slimmedJets can correctly re-calibrate the JEC for the PU jet ID.

I have tested that the PU jet ID is exactly the same when:
1. Running on AOD:
https://github.com/ahinzmann/jetScripts/blob/master/calculate_puid_from_AOD.py
https://github.com/ahinzmann/jetScripts/blob/master/read_AOD.py
2. Running on MiniAOD:
https://github.com/ahinzmann/jetScripts/blob/master/calculate_puid_from_MiniAOD.py
https://github.com/ahinzmann/jetScripts/blob/master/read_MiniAOD.py
3. Running on MiniAOD and create an updated pat jet collection with new JEC and new PU jet ID userfloat
https://github.com/ahinzmann/jetScripts/blob/master/update_jets_from_MiniAOD.py
https://github.com/ahinzmann/jetScripts/blob/master/read_updatedMiniAOD.py

**Comment by cvuosalo** [comment]
Is there any reason to keep this PR open still?

**Comment by cvuosalo** [comment]
@cmsbuild please test
We are not planning to approve this PR, but it shouldn't hurt to test it.

**Comment by slava77** [comment]
-1
tests have passed and differences appear only in \*MVAPU\* \*PUJID\* plots, as expected.
The tentative agreement was to close this PR once we see that it works.
It will be used separately for user recipes.

**Comment by ahinzmann** [comment]
many thanks! this can be closed now.

**Pull request closed without merging**